ios/Classes/SwiftHmssdkFlutterPlugin.swift

```diff
@@ -98,136 +98,215 @@ public class SwiftHmssdkFlutterPlugin: NSObject, FlutterPlugin, HMSUpdateListene
             
             // MARK:  Room Actions
             
-        case "build":
-            build(call, result)
-            
-        case "preview":
-            preview(call, result)
-            
-        case "join":
-            join(call, result)
-            
-        case "leave":
-            leave(result)
-            
-        case "get_room":
-            getRoom(result)
-            
-        case "get_local_peer":
-            getLocalPeer(result)
-            
-        case "get_remote_peers":
-            getRemotePeers(result)
-            
-        case "get_peers":
-            getPeers(result)
+        case "build","preview","join","leave","get_room","get_local_peer","get_remote_peers","get_peers":
```

Comment:
"build","preview","join","leave" should be in one function
"get_room","get_local_peer","get_remote_peers","get_peers" in another